resources/assets/js/components/JobReview/JobReviewPage.tsx

```diff
@@ -73,11 +74,11 @@ const JobBuilderReviewPage: React.FunctionComponent<
 
   const handleReturn = (): void => {
     // Go to Previous page
-    window.location.href = jobBuilderSkills(locale, jobId);
+    navigate(jobBuilderSkills(locale, jobId));
   };
   const handleContinue = (): void => {
     // Continue to next page
-    window.location.href = managerJobIndex(locale);
+    navigate(managerJobIndex(locale));
```

Comment:
I guess this should be reverted to preserve the right functionality?

Reply:
Yes, looks like I got over-enthusiastic with the navigate()'s